src/main/java/Duke.java

```diff
@@ -1,10 +1,87 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        int numOfTasks = 0;
+        // parses the user input
+        while (true) {
+            String input = in.nextLine();
+            if (input.equals("bye")) {
```

Comment:
You may want to simplify the code by using case statements. You can make your code more organized that way.